compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaTupleTypeSymbol.java

```diff
@@ -61,6 +65,29 @@ public List<TypeSymbol> memberTypeDescriptors() {
         return this.memberTypes;
     }
 
+    @Override
+    public List<TupleMemberTypeSymbol> tupleTypeMembers() {
+        if (this.tupleMembers != null) {
+            return this.tupleMembers;
+        }
+
+        TypesFactory typesFactory = TypesFactory.getInstance(this.context);
+
+        List<BTupleMember> tupMembers = ((BTupleType) this.getBType()).getTupleMembers();
+        this.tupleMembers = new ArrayList<>();
+
+        for (BTupleMember tupMember : tupMembers) {
+            BVarSymbol bSymbol = tupMember.symbol;
+            this.tupleMembers.add(new BallerinaTupleMemberSymbol(
+                    context,
+                    bSymbol.getType().name.getValue(),
+                    bSymbol,
+                    typesFactory.getTypeDescriptor(bSymbol.getType())));
```

Comment:
`bSymbol.getType()` can be extracted out to a variable.

Reply:
+1